src/features/onboarding/import/ImportPrivateKey.tsx

```diff
@@ -131,38 +129,25 @@ const ImportPrivateKey = () => {
     if (navigation.canGoBack()) {
       navigation.goBack()
     } else if (hasAccounts) {
-      navigation.replace(
-        l1Network === 'helium' ? 'HomeNavigator' : 'TabBarNavigator',
-      )
+      navigation.replace('TabBarNavigator')
     } else {
       navigation.replace('OnboardingNavigator')
     }
-  }, [hasAccounts, navigation, l1Network])
+  }, [hasAccounts, navigation])
 
   const onImportAccount = useCallback(() => {
     if (hasAccounts) {
-      if (l1Network === 'helium') {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        navigation.replace('HomeNavigator', {
```

Comment:
We should remove `HomeNavigator: undefined` from `src/navigation/rootTypes.ts`

Reply:
True let me do that